qemu/tests/cfg/mq_enabled_chk.cfg

```diff
@@ -0,0 +1,13 @@
+- mq_enabled_chk:
+    only Linux
+    only virtio_net
+    no RHEL.3 RHEL.4 RHEL.5
+    no RHEL.6.0 RHEL.6.1 RHEL.6.2 RHEL.6.3 RHEL.6.4 RHEL.6.5
+    no Host_RHEL.m5, Host_RHEL.m6
+    virt_test_type = qemu
+    type = mq_enabled_chk
+    vhost = "vhost=on"
+    image_snapshot = yes
```

Comment:
no need to use "vhost=on", it is default .

Reply:
I think better keep `vhost=on` here, in case someone change the default configuration later.